### DISCUSSION TITLE
how to do batch print

### DISCUSSION THREAD

**Discussion by cxc1357** [PDConSec/vsc-print#278] [Q&A] [answered]
like print all files in a folder to one pdf

**Comment by PeterWone** [comment, marked as answer]
Be aware that I am using Edge on Windows for these screen snaps. Browsers and operating systems vary but all Chrome derivative browsers can do this.

Right-click the folder and choose Print from the context menu

![image](https://github.com/PDConSec/vsc-print/assets/5498936/15c5ec70-973f-4ec7-a889-1ae3717efea6)

This will produce a combined print job like so

![image](https://github.com/PDConSec/vsc-print/assets/5498936/7d5409f9-b557-4cd7-a430-06563aa8c7ff)

Now choose PDF from the printers

![image](https://github.com/PDConSec/vsc-print/assets/5498936/b123be9d-3301-4c50-aa8e-802b137c26f2)

Notice that there are _two_ PDF options in this list. Use the first one, it's the direct Chrome PDF exporter. The other one is a virtual printer provided by Microsoft. It's handy for apps that can't do their own PDF export but I think the direct export produces better results.

After you select Save as PDF the dialog changes and you can click Save.

![image](https://github.com/PDConSec/vsc-print/assets/5498936/88aa5aba-c356-4e2a-87f0-2683cf63be20)

At this point you'll get whatever Save As dialog your operating system provides for you to specify the file name and location.

**Comment by cxc1357** [comment]
It works, thanks a lot !
